<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RMTS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="StartWidget" name="pageStart"/>
      <widget class="MotorDataWidget" name="pageRawDataMotorInfo"/>
      <widget class="FireWidget" name="pageFire"/>
      <widget class="ResultsWidget" name="pageResults"/>
      <widget class="PreferencesWidget" name="pagePreferences"/>
      <widget class="CalibrationWidget" name="pageCalibration"/>
      <widget class="TransducerEditorWidget" name="pageEditTransducer"/>
      <widget class="RecvResultsWidget" name="pageRecvResults"/>
      <widget class="CalibrationSetupWidget" name="pageCalibrationSetup"/>
      <widget class="AboutWidget" name="pageAbout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResultsWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.resultsWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StartWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.startWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FireWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.fireWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MotorDataWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.motorDataWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreferencesWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.preferencesWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransducerEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.transducerEditorWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecvResultsWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.recvResultsWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalibrationSetupWidget</class>
   <extends>QWidget</extends>
   <header>ui.calibrationSetupWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalibrationWidget</class>
   <extends>QWidget</extends>
   <header>ui.calibrationWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.aboutWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
